creator/events/migrations/0022_add_referral_token.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.26 on 2022-01-27 20:22
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('referral_tokens', '0003_add_referral_token'),
```

Comment:
```suggestion

```
So the tests are failing bc its trying to setup the test db and can't find the migration `0003_add_referral_token`. Bc you deleted it (as you should have) :)

If you remove this line everything will work